### ISSUE TITLE
One way sync option (read only)

### ISSUE THREAD

**Issue by nathan-ed** [orgzly-revived/orgzly-android-revived#439]
I use my phone to keep track of the notes I write on my computer. I would like to be able to have only a one-way sync option, so that the app only keeps track of a local folder without writing anything to it, updating the content of the app according to the local folder.

The current features do not allow this, since whenever a note is deleted on my computer and synced to my phone, the note still appears orgzly and is not removed. 

Thank you so much for the great work 😀

**Comment by amberin** [comment]
If remote deletions is your main issue, an option to immediately delete notebooks in such a scenario should be quite easy to add.

**Comment by nathan-ed** [comment]
Yes, something like a toggle option which says one way sync from a depository. The app would be in view only mode. I don't know if what I am saying makes sense.

**Comment by amberin** [comment]
Your request makes sense, although I suspect it is quite niche. I don't know how much work it would require, and if there is big enough of a demand to justify it. 

So far, you have only described your needs with respect to a remote deletion scenario. The toggle that I talked about would only alter that behavior, but nothing else.

Please describe why this would not suffice to satisfy your needs, so that we can understand them better.

**Comment by amberin** [comment]
Wait, are we talking about remotely deleted *notes* or *notebooks*?

**Comment by nathan-ed** [comment]
Ok, here is the complete setup. I have a syncthing between my phone and a server where my org files are stored. The sync from the server to the phone is one-way, so read only on the phone. 

I therefore have a local copy of my org file on my phone and want to view them in orgzly. To do so, I use the local depository option and open a directory and all its subrectories in orgzly. This works very well. Even if i accidentally edit a file, the syncing will overwrite the modification, so no issue there. The problem arises when I delete a file on the server, the file gets synced on the phone, but orgzly still keeps the old version of the file and saves it back to the directory.

I do this, since it is not convenient for me to edit org files on my phone and also because I ran into many conflicts and do not want to spend any time having to deal with that. I an mainly on my computer, but want to access all the info conveniently when on the move.

Thank you for your prompt replies and the effort you out into this app!

**Comment by amberin** [comment]
Ok, great, then we are talking deleted files/notebooks, as I thought. The current behavior is that they get de-linked from the repository, and not automatically re-linked. But it makes sense that some users may want them to get automatically deleted in Orgzly. This would be easy to add as an opt-in toggle, optionally with a notification when it happens.

**Comment by nathan-ed** [comment]
That is great, thanks for your replies!
